src/components/shell/shell.tsx

```diff
@@ -13,7 +13,9 @@ import { CSS, SLOTS } from "./resources";
  * @slot footer - A slot for adding footer content. This content will be positioned at the bottom of the component.
  * @slot panel-start - A slot for adding the starting `calcite-shell-panel`.
  * @slot panel-end - A slot for adding the ending `calcite-shell-panel`.
- * @slot center-row - A slot for adding content to the center row.
+ * @slot panel-top - A slot for adding the top `calcite-shell-panel`.
+ * @slot panel-bottom - A slot for adding the bottom `calcite-shell-panel`.
+ * @slot center-row - A slot for adding the bottom `calcite-shell-panel`. (Deprecated - use `panel-bottom` instead)
```

Comment:
I think this should start with deprecated for consistency.

```
 * @slot center-row - [Deprecated] use "panel-bottom" instead. A slot for adding content to the center row.
```

Reply:
Updated - @geospatialem this is the correct way to format "deprecated" slots right? We don't have any other current cases. Would this be a breaking change even though it's a "doc only" deprecation?

Reply:
We can verify in the next doc update with `1.3.0`, but it seems to be on par with what we had prior to 1.0. 👍🏻 

It looks like any text with `[deprecated]` mentioned receives the deprecation chip on the docs site (lower/upper case character combinations shouldn't matter). 

So `[Deprecated]` seems like a good pattern, and is what we had as of late November 2022 in [shell.txt](https://github.com/Esri/calcite-components/pull/5819/files#diff-b5b8c066968e2720736cedb2d71b8f55fa99bae5fbb8d8b28729cb47212de564) with `[DEPRECATED]` - so pick your poison with lower/upper case combinations.